images-dist/modules/sonataflow/common/scripts/added/quarkus-mvn-plugin.sh

```diff
@@ -24,7 +24,7 @@
 set -e
 
 # Default version variables
-quarkus_version="${QUARKUS_PLATFORM_VERSION}"
+quarkus_version="${QUARKUS_VERSION}"
```

Comment:
When we add a regular io.quarkus extension, the quarkus version must be used, and not the quarkus platform version.

see: https://github.com/kubesmarts/osl-images/pull/70/files#diff-89d825f550fb18cf8f2ea2b25feadde7b095e0c6b450cbe9f9a1250652137904R46